src/Client/SignatureVerifier.php

```diff
@@ -58,7 +57,7 @@ public function checkSignatures(MetadataBase $metadata): void
         $needVerified = $role->getThreshold();
         $verifiedKeySignatures = [];
 
-        $canonicalBytes = JsonNormalizer::asNormalizedJson($metadata->getSigned());
```

Comment:
So the only non-test use I see of `\Tuf\JsonNormalizer::asNormalizedJson. Could we just do:

`$keyCanonicalForm = json_encode(JsonNormalizer::rKeySort($keyCanonicalStruct), JSON_UNESCAPED_SLASHES);`
or something and just remove asNormalizedJson

Reply:
Yup, I was considering something like this. Thought it might be out of scope, but if you're +1, then I can do that.

Reply:
sure

Reply:
OK, I've created a new `CanonicalJsonTrait` which is the successor of `JsonNormalizer`. Metadata objects now have a `toCanonicalJson()` method.